app/helpers/blueberry_cms/pages_helper.rb

```diff
@@ -0,0 +1,19 @@
+module BlueberryCMS
+  module PagesHelper
+    def nested_set_options(class_or_item, mover = nil)
+      if class_or_item.is_a? Array
+        items = class_or_item.reject { |e| !e.root? }
```

Comment:
Jako fakt? reject !root?
Není varianta select root? jednodušší
```ruby
items = class_or_item.select(&:root?)
````
Otázkou taky je, jestli mít v applikaci víc rootů, já bych to osobně zakázal. Proč taky?